distribution/docs/src/main/resources/content/_dependency-list/ddf-dependency-list.adoc

```diff
@@ -0,0 +1,387 @@
+:title: Dependency List
+:type: appendix
+:status: published
+:parent: ${ddf-branding} Dependency List
+:children: none
+:order: 00
+:summary: ${ddf-branding} ${project.version} Dependency List.
+
```

Comment:
What happened to the note about this list is automatically generated?

Reply:
that's added by the `intro-contents` file. That part didn't change, but it's part of why this PR needs to be merged with the docs changeover.

Reply:
It might be worth adding a code comment in this file or a note in the summary so developers immediately know that this file doesn't need to be updated with dependency changes.